### DISCUSSION TITLE
Cancel vs Close in a modal

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#512] [General] [answered]
What is the purpose of Cancel vs  Close icon in the top right in a modal? For one of our use cases, we can’t truly cancel after user clicks on the primary action in the modal so we are considering disabling the cancel button as it will be misleading. This leaves us wondering what to do with the Close button on the top right. Should we disable it? Would leaving it active be confusing to the users of the purpose of Close?

**Comment by aayushpi** [comment]
Could you provide a screenshot @pguptaDS ?

**Comment by corinne-nagel** [comment]
Hey @pguptaDS can you please provide a screenshot so we can give you accurate feedback? Thanks!

**Comment by aayushpi** [comment, marked as answer]
Closed due to inactivity.
